lib/Home.dart

```diff
@@ -156,17 +156,13 @@ class _HomeState extends State<Home> {
       body: WatchBoxBuilder(
         box: Hive.box('pdfs'),
         builder: (context, pdfsBox) {
-          if (pdfsBox
-              .getAt(0)
-              .length == 0) {
+          if (pdfsBox.length == 0) {
```

Comment:
Rather than checking the length of pdfsBox here and in the starred screen, check if the code is actually adding some empty list at the '0'th position of both the boxes in main.dart. Reason - 

The saving code for the PDFs is written in such a way that it puts the list of lists on the 0th position and not adds them so we need that 0th position to be actually present to put something there.
```dart
    final List<dynamic> files = Hive.box('pdfs').getAt(0) as List<dynamic>;
    final now = DateTime.now();
    final String formatter = DateFormat('yMd').format(now);
    files.add([filePath, formatter]);
    Hive.box('pdfs').putAt(0, files);
```